src/main/java/br/codenation/hospital/manager/model/ItemEstoque.java

```diff
@@ -0,0 +1,16 @@
+package br.codenation.hospital.manager.model;
+
+import lombok.*;
```

Comment:
This is a bad pratice (importing everything from a package). If you are using Intellij, install the google-java-format plugin and change the imports!